src/app/App.tsx

```diff
@@ -13,7 +13,7 @@ import { MAAS_UI_ID } from "./constants";
 
 import Routes from "app/Routes";
 import Footer from "app/base/components/Footer";
-import Header from "app/base/components/Header";
+import GlobalSideNav from "app/base/components/GlobalSideNav";
```

Comment:
The previous implementation can be now deleted (`app/base/components/Header`)

Reply:
Actually, let's keep the `Header` for now until we decide what to do with the login screen - we might need to display it there until we implement the collapsed state.

Reply:
Yup, as discussed, let's display the existing Header for the logged out state. Could you look into this @ndv99 please

Reply:
https://warthogs.atlassian.net/browse/MAASENG-1311 @ndv99